app/views/activities/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="main-page">
+  <%= form_with model: @activity, url: user_category_activities_path(current_user, @category),  method: :post do |form| %>
+    <% if @activity.errors.any? %>
+      <div style="color: red">
+        <h2><%= pluralize(@category.errors.count, "error") %>
+          prohibited Transaction from being saved:</h2>
+
+        <ul>
+          <% @activity.errors.each do |error| %>
+            <li><%= error.full_message %></li>
+          <% end %>
+        </ul>
+      </div>
+    <% end %>
+    <div class="form">
+    <div class="field">
+      <%= form.text_field :name, placeholder: "Name", class:"input" %>
+    </div>
+
+    <div class="field">
+      <%= form.number_field :amount, placeholder: "Amount", class:"input" %>
+    </div>
+
+    <div class="field">
+      <%= form.collection_select :category_id,
+                             current_user.categories,
+                             :id,
+                             :name,
+                             include_blank: "Select category", class:"input" %>
+    </div>
+
+    <div class="actions">
+      <%= form.submit class:"btn" %>
+    </div>
+  <% end %>
+  </div>
```

Comment:
- [ ] The new activity form should have a back button "<" that takes me back to the activities page.

  ![image](https://github.com/Kaghenimbale/budget-app/assets/36542986/a0785097-ab22-4b3d-9f96-21f01eb65682)